GettingStarted.html

```diff
@@ -30,6 +30,18 @@ <h1 class="h1Other">Getting Started</h1>
 
   <div class="main">
     <div>
+      <h2>Setting Up a New Event in Cleus</h2>
```

Comment:
Nice job building out these secondary pages. I would consider cleaning up your file directory and using a `pages` folder or something so the root level (where index.html is) isn't quite so cluttered. Eventually, if you have a lot of pages, you may have subfolders grouping them even further - "how to", "admin", "marketing", etc.